tutorials/fundamentals/spread.md

```diff
@@ -0,0 +1,36 @@
+JavaScript's `spread` operator, `...`, is a handy way to pass 
+multiple values from an array, object, or anything you can iterate
+through to a function or variable assignment. It shines especially
+when you want to combine two or more arrays. Instead of having to
+use `push`, `splice`, `concat`, and the kitchen sink,  `...` makes
+it less of a headache:
+
+```javascript
+[require:Fundamentals spread-assignment]
+```
+
+**Note:** Avoid using `...` with multidimensional arrays.
+Modifying that kind of an array can lead to problems like
+shown below:
+
+```javascript
+[require:Fundamentals spread-carefully]
+```
+
+# Spread with Functions
+
+Using `...`, it reduces the chances of headaches arising when passing
```

Comment:
"The spread operator also lets you call a function with the contents of an array as parameters, similar to `apply()`."